test/contracts/big-floats/big-floats.c

```diff
@@ -147,135 +134,118 @@ void BigFloatCmpTest() {
 }
 
 void BigFloatSignTest() {
-    int reps, bigFloatHandle, result;
-    reps = int64getArgument(0);
-    bigFloatHandle = bigFloatNewFromFrac(0,1);
-    for (int i = 0; i < reps; i++)
-    {
+    int result;
+    int reps = int64getArgument(0);
+    int bigFloatHandle = bigFloatNewFromFrac(0,1);
+    for (int i = 0; i < reps; i++) {
         bigFloatGetArgument(i+1,bigFloatHandle);
         result = bigFloatSign(bigFloatHandle);
     }
     int64finish(result);
 }
 
 void BigFloatCloneTest() {
-    int reps, bigFloatHandle, result;
-    reps = int64getArgument(0);
-    bigFloatHandle = bigFloatNewFromFrac(0,1);
+    int result;
```

Comment:
The variable `result` doesn't seem to be used in this function.

Reply:
this has been solved